### PR TITLE
Task #4077 update view admin-user and layout admin master

### PR DESCRIPTION
**Pull request by chinn2206**
update view admin-user and layout admin master

### CODE REVIEW

**Comment by chinn2206** [comment]
redmine: https://edu-redmine.framgia.vn/issues/4077

**Comment by chinn2206** [comment]
reject

**Comment by chinn2206** [comment]
redmine: https://edu-redmine.framgia.vn/issues/4077

File: webpack.mix.js [hunk 1]
```diff
@@ -13,3 +13,8 @@ const mix = require('laravel-mix');
 
 mix.js('resources/js/app.js', 'public/js')
    .sass('resources/sass/app.scss', 'public/css');
+   
+mix.styles([
+	'public/css/app.css',
```
Comment (BunnyPi04) [inline webpack.mix.js]: tab

**Comment by BunnyPi04** [comment]
commented

**Comment by chinn2206** [comment]
ready

**Pull request merged**